src/python/src/grpc/_adapter/_blocking_invocation_inline_service_test.py

```diff
@@ -43,4 +43,4 @@ class BlockingInvocationInlineServiceTest(
 
 
 if __name__ == '__main__':
-  unittest.main()
+  unittest.main(verbosity=2)
```

Comment:
I've never before seen "verbosity=2" in a call to "unittest.main()". What does it do?